snippets/csharp/query_points.cs

```diff
@@ -55,3 +55,9 @@
 	usingVector: "colbert",
 	limit: 10
 );
+
+// Random sampling (as of 1.11.0)
+await client.QueryAsync(
+    collectionName: "{collection_name}",
+    query: new SampleQuery { Sample=Sample.Random }
```

Comment:
```suggestion
    query: Sample.Random
```
Should work because of this [implicit helper conversion](https://github.com/qdrant/qdrant-dotnet/blob/eb90d2b5e0c661f4aaa888b1a1860ec154fd8cf5/src/Qdrant.Client/Grpc/Query.cs#L43-L48).